packages/platform-sdk/src/contracts/coins/transaction.ts

```diff
@@ -149,4 +136,6 @@ export interface EntityUpdateInput extends TransactionInput {
 	};
 }
 
-export type TransactionInputs = Record<string, any>;
+export type TransactionInputs = Record<string, any> & {
```

Comment:
Is this & intentional? What does it mean?

Reply:
https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html#intersection-types